src/msw/webview_edge.cpp

```diff
@@ -588,6 +588,11 @@ void wxWebViewEdge::SetZoom(wxWebViewZoom zoom)
     m_impl->m_webViewController->put_ZoomFactor(zoom_factor);
```

Comment:
This is pretty minor, but I think this should call our `SetZoomFactor()` now.

Reply:
Will do